pyproject.toml

```
@@ -5,10 +5,13 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "arches-controlled-lists"
 readme = "README.md"
-authors = []
-license = {text = "GNU AGPL3"}
+authors = [
+    {"name" = "Arches Project"}
+]
+license = {text = "AGPL-3.0-or-later"}
```

At some point the old syntax is going to not work for uploading
```suggestion
license = "AGPL-3.0-or-later"
```